### ISSUE TITLE
[FEATURE] Accessibility Alt Text for Images

### ISSUE THREAD

**Issue by mirabtellegen** [tabler/tabler#1761]
## Summary

On the [Tabler documentation site](https://tabler.io/docs/getting-started), many images are not tagged with alt-text or aria labels. This poses an accessibility issue, as these images are not accessible without being able to visually understand the context and importance of the image. Some have good alt text or aria labels, and some have present labels but which are not descriptive.

## Background

Per the ADA, sites should be accessible to all users. Those who use screen readers or other accessibility tools may rely on alt text describing images or graphics. This alt-text requirement is codified in the [WCAG 2.0 ](https://www.w3.org/TR/WCAG20/) from 2008, in [Section 1.1.1](http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html) which states that any image or graphic other than those which are purely decorative or formatting should have alternative text which describes all information a user might glean from it visually.

## Aria Documentation Links

For image items, html alt-text would be in the form of an `alt` tag.

For non-image items, alt-text can be added with `aria-label` attributes. More documentation on Aria can be found [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label).

```
<a class="logo active" aria-label="Tabler Logo" href="/"></a>
```

## Itemized Issues

On the site, below are the images and potential alt-text.

No alt-text or aria-label: 

1. Potential alt-text: "Screencap of tabler site showing global traffic and statistics"
<img src="https://github.com/tabler/tabler/assets/145612743/5759401f-ce56-4cc2-9edc-95c3d2c2916c" width=200>

2. Potential alt-text for each logo: "Chrome logo", "Firefox logo", etc.
<img width="106" alt="browser logos on tabler site" src="https://github.com/tabler/tabler/assets/145612743/bb7423df-7027-44ed-9356-a5a55f3620f2">

3. Potential alt-text: "Tabler icon and PDF icon merging with npm command to install"
<img src="https://github.com/tabler/tabler/assets/145612743/ac6a8795-2bcd-4858-98c3-559b6febd231" width=200>

**Comment by kevinpapst** [comment]
Some images have been tagged. 
Problem with issues like yours are, that there is no clear goal and therefor no clear point at which they can be closed.

What about closing this one, as #1786 was merged and then you can re-test and create a new issue e.g. with a checkbox list of the 10 most pressing WCAG problems?
